docs/build/sdk/introduction.md

```diff
@@ -2,6 +2,10 @@
 
 ![sdk version](https://img.shields.io/github/v/tag/zeitgeistpm/tools?label=sdk)
 
+The Zeitgeist SDK is written in `Typescript` and can be found here [@zeitgeistpm/sdk](https://github.com/zeitgeistpm/tools/tree/main/packages/sdk) You can do the following to install it directly into your javascript based project:
+
+`npm i @zeitgeistpm/sdk` or `yarn add @zeitgeistpm/sdk`
```

Comment:
~~~suggestion
The Zeitgeist SDK is written in `Typescript` and can be found here:
[@zeitgeistpm/sdk](https://github.com/zeitgeistpm/tools/tree/main/packages/sdk).
You can do the following to install it directly into your javascript based
project:

```
npm i @zeitgeistpm/sdk` or `yarn add @zeitgeistpm/sdk
```
~~~